.github/workflows/lint.yml

```diff
@@ -1,16 +0,0 @@
-name: Lint
```

Comment:
To reviewers: ignore this change! It's done in order to prevent ESLint errors from showing up on the PR page.